ember-client/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -186,6 +188,31 @@ final class EmberClientBuilder[F[_]: Async] private (
             }
           }
         } yield responseResource._1
+
+      def unixSocketClient(
+          request: Request[F],
+          address: UnixSocketAddress): Resource[F, Response[F]] =
+        Resource
+          .eval(EmberConnection(ClientHelpers.unixSocket(request, address, tlsContextOpt)))
```

Comment:
More of a question, there's no advantage to using a keypool here?

Reply:
This one I'm uncertain about. You can get all the same advantages but comes with another pool of state to drag around, and I doubt this will often even be used. I think it can be added later the same as the other one if we decide we want pooling.

Reply:
Am I crazy to wonder if there's some way to re-use the current keypool? That's what I was initially imagining.

Reply:
No, thats a totally different resource.

Reply:
The KeyPool maps `RequestKey` to `EmberConnection[F]`, right? And both the ordinary sockets and unix sockets use `EmberConnection[F]`, correct? So can't we just add the unix socket path or something to the `RequestKey` to make this work? Apologies if I'm missing something.

Reply:
But why? The state is fully decoupled. Its equivalent to a fully different pool.

Reply:
> You can get all the same advantages but comes with another pool of state to drag around

This seemed like reason enough to me. Just a thought.

Reply:
That was my reason to not use a pool. Not to complicate the other one which is already a significant headache to think through today.

If you are insistent I use the pool I will make a new one. I just don't see the need for a first effort.

Reply:
> Not to complicate the other one which is already a significant headache to think through today.

Nope, not insistent at all, you seemed on the fence about this so I asked. Furthermore, I didn't appreciate the weight of messing with the current pool, I agree better let that sleeping dog lie :)